### ISSUE TITLE
[A11y] Homepage - Parsing error

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1909]
### Operating System

Windows

### Browser

Edge

### Affected page

Homepage

### Admin user?

Yes

### Issue Description

There are a lot of nested divs inside labels, which is a parsing failure.  I can't find the actual control on the page that makes this show up, so I'm thinking this HTML might actually be a mistaken widget that doesn't need to be here in the first place (though if it's in use elsewhere it would still need to be fixed).

Here's an example of the HTML:

```html
          <div class="select-worktype">
            <label>
              <input type="radio" name="payload_concern" value="Image" data-single="/concern/images/new?locale=en" data-batch="/batch_uploads/new?locale=en&amp;payload_concern=Image">
              <div class="select-work-icon">
                <span class="fa fa-file-text-o"></span>
              </div>
              <div class="select-work-description">
                <h3 class="work-type-title">Image</h3>
                <p>Image works</p>
              </div>
            </label>
          </div>
```

### Suggested Fix

Just make the `<p>Image works</p>` into a label and associate it with the radio button normally (using the `for` attribute).

The label seems wrong in this case anyway, and should just be wrapping the "Image works" bit, I think.  Wrapping the whole thing means the label for that radio button is "Image Image works", which makes less sense.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

The WCAG scan will need to be used to verify nothing is broken on this page.  This is most easily done with the Accessibility Insights bookmarklets.

Source will need to be inspected to verify the label is properly associated, since that will change when fixing the invalid nesting.

**Comment by jechols** [comment]
Well, the divs are no longer an issue, but unfortunately the new HTML chosen isn't actually valid. Spans are only allowed to have "phrasing content": https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content

Failure screenshot:

![image](https://user-images.githubusercontent.com/2157974/152832713-7c5a1b74-43ca-488c-95a3-6d29c638c3d2.png)

**Comment by azle** [comment]
Updated 1/6/2024 FIXED

Looks like the code has been updated:
<img width="554" alt="Screenshot 2025-01-06 at 10 28 10 AM" src="https://github.com/user-attachments/assets/6fa37d8e-a078-4c45-9213-18a4abae2f71" />
